README.md

```diff
@@ -38,11 +38,13 @@ Jekyll uses the [Liquid template engine](http://liquidmarkup.org/) for templatin
 
 You can use [GFM](http://kramdown.gettalong.org/parser/gfm.html) fenced code blocks for JavaScript; for example:
 
+````md
 ```js
 var express = require('express')
 var app = express()
 app.listen(3000)
 ```
+````
 
 The default GitHub Pages syntax highlighting has been disabled in `_config.yml to allow highlighting with prism.js.
```

Comment:
As long as you're at it, could you please fix this unrelated issue --- add the terminating backtick... s/be `_config.yml` .. thx!